pkg/client/image.go

```diff
@@ -19,15 +19,16 @@ func (c *client) ImageTag(ctx context.Context, imageName, tag string) error {
 	if apierrors.IsNotFound(err) {
 		return err
 	}
-	image.Tags = []string{tag}
 	tagResult := &apiv1.ImageTag{}
 	err = c.RESTClient.Post().
 		Namespace(image.Namespace).
 		Resource("images").
 		Name(image.Name).
 		SubResource("tag").
 		Body(&apiv1.ImageTag{
-			Tags: []string{tag},
+			Tags:       []string{tag},
+			ObjectMeta: image.ObjectMeta,
```

Comment:
Why are you copying the ObjectMeta? For sure you shouldn't copy the TypeMeta because that will set the TypeMeta from image to imagetag which are different types.

Reply:
I had thought the reasoning for that weird err might have been something with the image version mismatching in object meta but that wasn't the case. removed both